pkg/spec/resources_transform.go

```diff
@@ -77,7 +75,7 @@ func createServices(app *App) ([]runtime.Object, error) {
 		}
 		svcs = append(svcs, svc)
 
-		// Generate ingress if "endpoint" is mentioned in app.Services.Ports[].Endpoint
+		// Generate ingress if "endpoint" is mentioned in deployment.Services.Ports[].Endpoint
```

Comment:
This will still be `app`, because of the following reasons:

- this is not tied to deployment in anyway
- user will still be considering it as app
- the error is user facing

Reply:
ack, changed